content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -196,6 +196,8 @@ and is likely to hurt functionality more than it improves performance.
 
 ### IPVS proxy mode {#proxy-mode-ipvs}
 
+{{< feature-state for_k8s_version="v1.35" state="deprecated" >}}
```

Comment:
There's some text below saying "At some point in the future, it is expected to be formally deprecated", so you need to update that.

Not sure if there should be any other reordering.

Reply:
Good catch, I just deleted that, since this line is saying that ipvs is deprecated.